website/common/script/index.js

```diff
@@ -77,6 +77,9 @@ api.updateStore = updateStore;
 import inAppRewards from './libs/inAppRewards';
 api.inAppRewards = inAppRewards;
 
+import setDebuffPotionItem from './libs/setDebuffPotionItem';
```

Comment:
is there a specific reason why the `getDebuffPotionItem ` file isn't added as well?

Reply:
I lead by principle -- make all as private as possible... so in server code I see only reasons to set item for user and incapsulate all knowledge about it inside set function. Of course I don't familiar with the project so it might be wrong...

Reply:
Yeah so that file used to be the gate to all the common code but on the client we no import only the necessary files but it's still being used in some places on the server so I'd say let's include and we'll remove it when we finally do some refactoring :)

Reply:
Ok, sure :)